man/systemd.network.xml

```diff
@@ -1676,6 +1676,23 @@
             then all options specified earlier are cleared. Defaults to unset.</para>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>EnableSourceRoutingRules=</varname></term>
+          <listitem>
+            <para>This option is used to configure source routing for an interface.
+            When combined with setting an explicit route table (with the <literal>RouteTable=</literal> option),
+            this causes routing policy rules to be automatically be created for the DHCP obtained address on the interface,
+            of the form "from $address lookup $routetable".</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>SourceRoutingRulesPriority=</varname></term>
+          <listitem>
+            <para>When <literal>EnableSourceRoutingRules=</literal> is enabled, this controls the priority of the routing
+            policy rules that are created.</para>
```

Comment:
please specify valid range, and whether smaller or larger values are more important. (alternatively point the reader to a relevant related option where this is explained)

Reply:
also, should say what the default is.